### PR TITLE
add missing Kafka log source in error

### PR DESCRIPTION
**Pull request by benzekrimaha**
Issue : BB-530

### CODE REVIEW

File: lib/queuePopulator/QueuePopulator.js [hunk 1]
```diff
@@ -350,7 +350,7 @@ class QueuePopulator {
                 break;
             default:
                 throw new Error("bad 'logSource' config value: expect 'bucketd,'" +
-                    ` mongo or 'dmd', got '${this.qpConfig.logSource}'`);
+                    ` mongo or 'kafka' or 'dmd', got '${this.qpConfig.logSource}'`);
```
Comment (ghost) [inline lib/queuePopulator/QueuePopulator.js]:
```suggestion
                throw new Error("bad 'logSource' config value: expect 'bucketd'," +
                    ` 'mongo', 'kafka' or 'dmd', got '${this.qpConfig.logSource}'`);
```

**Review by ghost — APPROVED** [review]

File: lib/queuePopulator/QueuePopulator.js (hunk 1 above)
Comment (ghost) [reply lib/queuePopulator/QueuePopulator.js]: Also the ticket says "_setupLogSources(): missing Kafka log source in the error we throw when no log source corresponds."
but it was here before right? Are we missing something?

**Review by francoisferrand — APPROVED** [review]

File: lib/queuePopulator/QueuePopulator.js (hunk 1 above)
Comment (benzekrimaha) [reply lib/queuePopulator/QueuePopulator.js]: yes it is already in the error message, what was meant in the ticket is only that kafka was not listed as one of the expected values from what I understood

**Pull request merged**